tests/protocol/test_extensions_goal_state.py

```diff
@@ -25,24 +25,3 @@ def test_create_from_vm_settings_should_assume_block_when_blob_type_is_not_valid
         extensions_goal_state = ExtensionsGoalStateFactory.create_from_vm_settings(1234567890, load_data("hostgaplugin/vm_settings-invalid_blob_type.json"))
         self.assertEqual("BlockBlob", extensions_goal_state.status_upload_blob_type, 'Expected BlockBob for an invalid statusBlobType')
 
-    def test_extension_goal_state_should_parse_requested_version_properly(self):
```

Comment:
These tests have also been giving me pain. I moved them to tests for parsing extensionsConfig/vmSettings